src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/EditApprenticeshipRequestToViewModelMapperTests.cs

```diff
@@ -313,6 +312,36 @@ public async Task IsLockedForUpdate_Is_Mapped_With_ApprenticeshipStatus_And_IsNo
             Assert.AreEqual(expectedIsLockedForUpdated, viewModel.IsLockedForUpdate);
         }
 
+        [TestCase(ApprenticeshipStatus.Paused, true)]
+        public async Task IsLockedForUpdate_And_IsLive_Is_Mapped_With_ApprenticeshipStatus_And_IsNotWithInFundingPeriod_Condition(ApprenticeshipStatus status, bool expectedIsLockedForUpdated)
```

Comment:
This test has been split out and renamed to a misleading name for no good reason? Should have just been a changed to the expected result in the above test case